hackathon_site/dashboard/frontend/src/components/teamDetail/TeamPendingOrderTable/TeamPendingOrderTable.test.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { getByTestId, render, screen } from "testing/utils";
+import { mockHardware, mockPendingOrdersInTable } from "testing/mockData";
+import TeamPendingOrderTable from "components/teamDetail/TeamPendingOrderTable/TeamPendingOrderTable";
+import { within, waitFor } from "testing/utils";
+import { fireEvent } from "@testing-library/react";
+
+describe("team pending order table", () => {
+    test("renders team pending order table", () => {
+        const { getByTestId } = render(<TeamPendingOrderTable />);
+        expect(screen.getByText("Requested Items")).toBeInTheDocument();
+
+        //loop through all pending orders
+        for (let i = 0; i < mockPendingOrdersInTable.length; i++) {
```

Comment:
just fyi you can do `mockPendingOrdersInTable.forEach((currentOrder, idx) => {})` because it's an array, most JS developers don't use normal for loops anymore bc the forEach() function is so much cleaner